### DISCUSSION TITLE
§No☆●◎□◆○◎★Ё Б Г Д Е 军营征集军兵

### DISCUSSION THREAD

**Discussion by Soldier11-ObsidianBarracks** [share121/inter-knot#1605] [其他]
前言
-----------------------------------------------------------------------------------------------
新艾丽都/新艾丽都在~~dwвсы Θ Ξ ΠыаоущшыΓ Δ Θ Ξihыd~~的历史中，被§№☆●◎□◆○◎★迫害许久，因此，我们§№☆●◎□◆○◎★Ё Б Г Д Е 军营诚邀各位加入
条件如下
----------------------------------------------------------------------------------
第一章
性别：男/女
男性身高160cm以上，女性身高158cm以上
***
体重符合下列条件且空腹血糖<7.0mmol/L的
（一）男性：17.5≤BMI<30,其中：17.5≤男性身体条件兵BMI<27；
（二）女性：17≤BMI<24。
BMI≥28须加查血液化血红蛋白检查项目
糖化血红蛋白百分比<6.5%，合格。
（BMI=体重（千克）除以身高（米）的平方）
***
颅脑外伤，颅脑畸形，颅脑手术史，脑外伤后综合症，不合格
颈部运动功能受限，斜颈，Ⅲ度以上单纯性甲状腺肿，乳腺肿瘤，不合格
骨、关节、滑囊疾病或损伤及其后遗症，骨、关节畸形，胸廓畸形，习惯性脱臼，颈、胸、腰椎骨折史，腰椎间盘突出，强直性脊柱炎，影响肢体功能的腱鞘疾病，不合格
下列情况合格：
（一）可自行矫正的脊柱侧弯；
（二）四肢单纯性骨折，治愈1年后，X线片显示骨折线消失，复位良好，无功能障碍及后遗症（条件兵除外）；
（三)关节弹响排除骨关节疾病或损伤，不影响正常功能的；
（四）大骨节病仅指、趾关节稍粗大，无自觉症状，无功能障碍（仅陆勤人员）；
（五）轻度胸廓畸形（条件兵除外）。

第六条 肘关节过伸超过15度，肘关节外翻超过20度，或虽未超过前述规定但存在功能障碍，不合格。
第七条 下蹲不全，两下肢不等长超过2cm，膝内翻股骨内髁间距离和膝外翻胫骨内踝间距离超过7cm（条件兵超过4cm），或虽未超过前述规定但步态异常，不合格。
轻度下蹲不全（膝后夹角≤45度），除条件兵外合格。
双足并拢不能完全下蹲，或勉强下蹲不稳者，可调整下蹲姿势（双足分开不超过肩宽），调整姿势后能完全下蹲或轻度下蹲不全者，陆勤人员合格（臀肌挛缩综合征、跟腱短、下肢关节病变等病理性原因除外）。

第八条 手指、足趾残缺或畸形，足底弓完全消失的扁平足，重度皲裂症，不合格。

第九条 恶性肿瘤，面颈部长径超过1cm的良性肿瘤、囊肿，其他部位长径超过3cm的良性肿瘤、囊肿，或虽未超出前述规定但影响功能和训练的，不合格。

第十条 瘢痕体质，面颈部长径超过3cm或影响功能的瘢痕，其他部位影响功能的瘢痕，不合格。

第十一条 面颈部文身，着军队制式体能训练服其他裸露部位长径超过3cm的文身，其他部位长径超过10cm的文身，男性文眉、文眼线、文唇，女性文唇，不合格。

第十二条 脉管炎，动脉瘤，中、重度下肢静脉曲张和精索静脉曲张，不合格。下肢静脉曲张，精索静脉曲张，条件兵不合格。

第十三条 胸、腹腔手术史，疝，脱肛，肛瘘，肛旁脓肿，重度陈旧性肛裂，环状痔，混合痔，不合格。

下列情况合格：
（一）阑尾炎手术后半年以上，无后遗症；
（二）腹股沟疝、股疝手术后1年以上，无后遗症；
（三）2个以下且长径均在0.8cm以下的混合痔。

第十四条 泌尿生殖系统疾病或损伤及其后遗症，生殖器官畸形或发育不全，单睾，隐睾及其术后，不合格。

下列情况合格：
（一）无自觉症状的轻度非交通性精索鞘膜积液，不大于健侧睾丸（条件兵除外）；
（二）无自觉症状的睾丸鞘膜积液，包括睾丸在内不大于健侧睾丸1倍（条件兵除外）；
（三）交通性鞘膜积液，手术后1年以上无复发，无后遗症；
（四）无压痛、无自觉症状的精索、副睾小结节，数量在2个以下且长径均在0.5cm以下；
（五）包茎、包皮过长（条件兵除外）；
（六）轻度急性包皮炎、阴囊炎。

第十五条 重度腋臭，不合格。轻度腋臭，条件兵不合格。

第十六条 头癣，泛发性体癣，疥疮，慢性泛发性湿疹，慢性荨麻疹，泛发性神经性皮炎，银屑病，面颈部长径超过1cm的血管痣、色素痣、胎痣和白癜风，其他传染性或难 以治愈的皮肤病，不合格。多发性毛囊炎，皮肤对刺激物过敏或有接触性皮炎史，手足部位近3年连续发生冻疮，条件兵不合格。

下列情况合格：
（一）单发局限性神经性皮炎，长径在3cm以下；
（二）股癣，手（足）癣，甲（指、趾）癣，躯干花斑癣；
（三）身体其他部位白癜风不超过2处，每处长径在3cm以下。

第十七条 淋病，梅毒，软下疳，性病性淋巴肉芽肿，非淋菌性尿道炎，尖锐湿疣，生殖器疱疹，以及其他性传播疾病，不合格。

（智械请参考第一章及第五章，二至三章为有机生命体的标准）

第二章 内科

第十八条 血压在下列范围，合格。
（一）收缩压≥90 mmHg，＜140 mmHg；
（二）舒张压≥60 mmHg，＜90 mmHg。

第十九条 心率在下列范围，合格。
（一）心率60～100次/分；
（二）心率50～59次/分或101～110次/分，经检查系生理性（条件兵除外）。

第二十条 高血压病，器质性心脏病，血管疾病，右位心脏，不合格。

下列情况合格：
（一）听诊发现心律不齐、心脏收缩期杂音的，经检查系生理性（条件兵除外）；
（二）直立性低血压、周围血管舒缩障碍（仅陆勤人员）。

第二十一条 慢性支气管炎，支气管扩张，支气管哮喘，肺大泡，气胸及气胸史，以及其他呼吸系统慢性疾病，不合格。

第二十二条 严重慢性胃、肠疾病，肝脏、胆囊、脾脏、胰腺疾病，内脏下垂，腹部包块，不合格。
下列情况合格：

（一）仰卧位，平静呼吸，在右锁骨中线肋缘下触及肝脏不超过1.5cm，剑突下不超过3cm，质软，边薄，平滑，无触痛、叩击痛，肝上界在正常范围，左肋缘下未触及脾脏，无贫血，营养状况良好；
（二）既往因患疟疾、血吸虫病、黑热病引起的脾脏肿大，现无自觉症状，无贫血，营养状况良好。

第二十三条 泌尿、血液、内分泌系统疾病，代谢性疾病，免疫性疾病，不合格。

第二十四条 艾滋病，病毒性肝炎，结核，流行性出血热，细菌性和阿米巴性痢疾，黑热病，伤寒，副伤寒，布鲁氏菌病，钩端螺旋体病，血吸虫病，疟疾，丝虫病，以及其他传染病，不合格。
下列情况合格：

（一）急性病毒性肝炎治愈后2年以上未再复发，无症状和体征，实验室检查正常；
（二）原发性肺结核、继发性肺结核、结核性胸膜炎、肾结核、腹膜结核，临床治愈后3年无复发（条件兵除外）；
（三）细菌性痢疾治愈1年以上；
（四）疟疾、黑热病、血吸虫病、阿米巴性痢疾、钩端螺旋体病、流行性出血热、伤寒、副伤寒、布鲁氏菌病，治愈2年以上，无后遗症；
（五）丝虫病治愈半年以上，无后遗症。

第二十五条 癫痫，以及其他神经系统疾病及后遗症，不合格。

第二十六条 精神分裂症，转换性障碍，分离性障碍，抑郁症，躁狂症，精神活性物质滥用和依赖，人格障碍，应激障碍， 睡眠障碍，进食障碍，精神发育迟滞，遗尿症，以及其他精神类疾病，不合格。

第二十七条 影响正常表达的口吃，不合格。

第三章 耳鼻咽喉科

第二十八条 听力测定双侧耳语均低于5m，不合格。
一侧耳语5m、另一侧不低于3m，陆勤人员合格。

第二十九条 眩晕病，不合格。

第三十条 耳廓明显畸形，外耳道闭锁，反复发炎的耳前瘘管，耳廓及外耳道湿疹，耳霉菌病，不合格。
轻度耳廓及外耳道湿疹，轻度耳霉菌病，陆勤人员合格。

第三十一条 鼓膜穿孔，化脓性中耳炎，乳突炎，以及其他难以治愈的耳病，不合格。鼓膜中度以上内陷，鼓膜瘢痕或钙化斑超过鼓膜的1/3，咽鼓管通气功能、耳气压功能及鼓膜活动不良，咽鼓管咽口或周围淋巴样组织增生，条件兵不合格。
鼓膜内陷、粘连、萎缩、瘢痕、钙化斑，条件兵合格。

第三十二条 嗅觉丧失，不合格。嗅觉迟钝，条件兵不合格。

第三十三条 鼻中隔穿孔，鼻畸形，重度肥厚性鼻炎，萎缩性鼻炎，重度鼻粘膜糜烂，鼻息肉，中鼻甲息肉样变，以及其他影响鼻功能的慢性鼻病，不合格。严重变应性鼻炎，肥厚性鼻炎，慢性鼻窦炎，严重鼻中隔偏曲，条件兵不合格。
不影响副鼻窦引流的中鼻甲肥大，中鼻道有少量粘液脓性分泌物，轻度萎缩性鼻炎，陆勤人员合格。

第三十四条 超过Ⅱ度肿大的慢性扁桃体炎，影响吞咽、发音功能难以治愈的咽、喉疾病，严重阻塞性睡眠呼吸暂停综合征，不合格。

第四章 眼科

第三十五条 任何一眼裸眼视力低于4.5，不合格。

任何一眼裸眼视力低于4.8，需进行矫正视力检查，任何一眼矫正视力低于4.8或矫正度数超过600度，不合格。
屈光不正经准分子激光手术（不含有晶体眼人工晶体植入术等其他术式）后半年以上，无并发症，任何一眼裸眼视力达到4.8，眼底检查正常，除条件兵外合格。
条件兵视力合格条件按有关标准执行。

第三十六条 色弱，色盲，不合格。
能够识别红、绿、黄、蓝、紫各单色者，陆勤人员合格。

第三十七条 影响眼功能的眼睑、睑缘、结膜、泪器疾病，不合格。
伸入角膜不超过2mm的假性翼状胬肉，陆勤人员合格。

第三十八条 眼球突出，眼球震颤，眼肌疾病，不合格。
15度以内的共同性内、外斜视，陆勤人员合格。

第三十九条 角膜、巩膜、虹膜睫状体疾病，瞳孔变形、运动障碍，不合格。
不影响视力的角膜云翳，合格。

第四十条 晶状体、玻璃体、视网膜、脉络膜、视神经疾病，以及青光眼，不合格。
先天性少数散在的晶状体小混浊点，合格。

第五章 口腔科

第四十一条深度龋齿超过3个，缺齿超过2个（经正畸治疗拔除、牙列整齐的除外），全口义齿及复杂的可摘局部义齿，重度牙周炎，影响咀嚼及发音功能的口腔疾病，颞颌关节疾病，唇、腭裂及唇裂术后明显瘢痕，不合格。
经治疗、修复后功能良好的龋齿、缺齿，合格。

第四十二条 中度以上氟斑牙及牙釉质发育不全，切牙、尖牙、双尖牙明显缺损或缺失，超牙合超过0.5cm，开牙合超过0.3cm，上下颌牙咬合到对侧牙龈的深覆牙合，反牙合，牙列不齐，重度牙龈炎，中度牙周炎，条件兵不合格。

下列情况合格：

（一）上下颌左右尖牙、双尖牙咬合相距0.3cm以内；
（二）切牙缺失1个，经固定义齿修复后功能良好，或牙列无间隙，替代牙功能良好；
（三）不影响咬合的个别切牙牙列不齐或重叠；
（四）不影响咬合的个别切牙轻度反牙合，无其他体征；
（五）错牙合畸形经正畸治疗后功能良好。

第四十三条 慢性腮腺炎，腮腺囊肿，口腔肿瘤，不合格。

第六章 妇科

第四十四条 闭经，严重痛经，子宫不规则出血，功能性子宫出血，子宫内膜异位症，不合格。

第四十五条 内外生殖器畸形或缺陷，不合格。

第四十六条 急、慢性盆腔炎，盆腔肿物，不合格。

第四十七条 霉菌性阴道炎，滴虫性阴道炎，不合格。

第四十八条 妊娠，不合格。

第七章 辅助检查

第四十九条 血细胞分析结果在下列范围，合格。

（一）血红蛋白：男性130～175g／L，女性115～150g／L；
（二）红细胞计数：男性4.3～5.8×1012／L，女性3.8～5.1×1012／L；
（三）白细胞计数：3.5～9.5×109／L；
（四）中性粒细胞百分数：40％～75％；
（五）淋巴细胞百分数：20％～50％；
（六）血小板计数：125～350×109／L。
血常规检查结果要结合临床及地区差异作出正确结论。血红蛋白、红细胞数、白细胞总数、白细胞分类、血小板计数稍高或稍低，根据所在地区人体正常值范围，在排除器质性病变的前提下，不作单项淘汰。

第五十条 血生化分析结果在下列范围，合格。

（一）血清丙氨酸氨基转移酶:男性9～50 U/L，女性7～40 U/L；
血清丙氨酸氨基转移酶，男性>50 U/L、≤60 U/L，女性>40 U/L、≤50 U/L，应当结合临床物理检查，在排除疾病的情况下，视为合格，但须从严掌握；

（二）血清肌酐：

酶法：男性59～104μmol/L，女性45～84μmol/L；
苦味酸速率法：男性62～115μmol/L，女性53～97μmol/L；
苦味酸去蛋白终点法：男性44～133μmol/L，女性70～106μmol/L；
（三）血清尿素：2.9～8.2mmol/L。

第五十一条 乙型肝炎表面抗原检测阳性，艾滋病病毒（HIV1+2）抗体检测阳性，不合格。

第五十二条 尿常规检查结果在下列范围，合格。

（一）尿蛋白：阴性至微量；
（二）尿酮体：阴性；
（三）尿糖：阴性；
（四）胆红素：阴性；
（五）尿胆原：0.1～1.0 Eμ／dl(弱阳性)。
尿常规检查结果要结合临床及地区差异作出正确结论。

第五十三条 尿液离心沉淀标本镜检结果在下列范围，合格。
（一）红细胞：男性0～偶见／高倍镜，女性0～3／高倍镜，女性不超过6个/高倍镜应结合外阴检查排除疾病；
（二）白细胞：男性0～3／高倍镜，女性0～5／高倍镜，不超过6个/高倍镜应结合外生殖器或外阴检查排除疾病；
（三）管型：无或偶见透明管型，无其他管型。
第五十四条 尿液毒品检测阳性，不合格。
第五十五条 尿液妊娠试验阴性，合格。
尿液妊娠试验阳性、但血清妊娠试验阴性，合格。

第五十六条 大便常规检查结果在下列范围，合格。
（一）外观：黄软；
（二）镜检：红、白细胞各0～2／高倍镜，无钩虫、鞭虫、绦虫、血吸虫、肝吸虫、姜片虫卵及肠道原虫。
大便常规检查，在地方性寄生虫病和血吸虫病流行地区为必检项目，其他地区根据需要进行检查。

第五十七条 胸部X射线检查结果在下列范围内，合格。
（一）胸部X射线检查未见异常；
（二）孤立散在的钙化点(直径不超过0.5cm)，双肺野不超过3个，密度高，边缘清晰，周围无浸润现象（条件兵除外）；
（三）肺纹理轻度增强(无呼吸道病史，无自觉症状；
（四）一侧肋膈角轻度变钝(无心、肺、胸疾病史，无自觉症状)。

第五十八条 心电图检查结果在下列范围内，合格。
（一）正常心电图；
（二）大致正常心电图。大致正常心电图范围按有关规定执行。

第五十九条 腹部超声检查发现恶性征象、病理性脾肿大、胰腺病变、肝肾弥漫性实质损害、肾盂积水、结石、内脏反位、单肾以及其他病变和异常的，不合格。

下列情况合格（第五至十一款，条件兵除外）：
（一）肝、胆、胰、脾、双肾未见明显异常；
（二）轻、中度脂肪肝且肝功能正常；
（三）胆囊息肉样病变，数量3个以下且长径均在0.5cm以下；
（四）副脾；
（五）肝肾囊肿和血管瘤单脏器数量3个以下且长径均在1cm以下；
（六）单发肝肾囊肿和血管瘤长径3cm以下；
（七）肝、脾内钙化灶数量3个以下且长径均在1cm以下；
（八）双肾实质钙化灶数量3个以下且长径1cm以下；
（九）双肾错构瘤数量2个以下且长径均在1cm以下；
（十）肾盂宽不超过1.5cm，输尿管不增宽；
（十一）脾脏长径10cm以下，厚度4.5cm以下；脾脏长径超过10cm或厚径超过4.5cm，但脾面积测量（0.8×长径×厚径）38cm2以下，排除器质性病变。

第六十条 妇科超声检查发现子宫肌瘤、附件区不明性质包块、以及其他病变和异常的，不合格。
下列情况合格：
（一）子宫、卵巢大小形态未见明显异常；
（二）不伴其他异常的盆腔积液深度不超过2cm；
（三）单发附件区、卵巢囊肿长径小于3cm。

第六十一条年龄18~40岁

第六十二条机动性，反应力较强

第五章

身高1.75m以上

1.1机械骨骼，关节完好

1.2计算数据芯片完好

1.3机械电路无短路情况

1.4外壳完好，材质需为马氏体硬化钢

1.5其他请参考第一章

请各位踊跃报名
======================================================================================

[报名地址](https://www.gfbzb.gov.cn/)

**Comment by sishui-buyin** [comment]
我去？哥们你来真的

**Reply by Soldier11-ObsidianBarracks** [reply]
加纳~（bushi）

**Comment by szlx2333** [comment]
8月10号下半年征兵就结束了，已过期

**Comment by Soldier11-ObsidianBarracks** [comment]
忘记哩，明年记得参加征兵（bushi
